src/core/feed-core.js

```diff
@@ -18,11 +18,14 @@ function getStickySqlString() {
       COALESCE(users.name, 'SYSTEM') as user_name,
       users.uuid as user_uuid,
       teams.name as team_name,
+      votes(feed_items) as votes,
```

Comment:
Function renamed to vote_score.

Compared to inline, the function mostly provides convenience as the same calculation is used multiple times.